llvm/include/llvm/CodeGen/SelectionDAGNodes.h

```diff
@@ -923,6 +923,7 @@ END_TWO_BYTE_PACK()
 
   /// Helper method returns the integer value of a ConstantSDNode operand.
   inline uint64_t getConstantOperandVal(unsigned Num) const;
+  inline uint64_t getAsConstantVal() const;
```

Comment:
Potentially confusing if you need a signed immediate, but that's already an issue with the existing helper

Reply:
Yes, I'm definitely inheriting that confusion. Would getAsZExtVal be better?

Reply:
I've gone ahead and renamed to getAsZExtVal instead and rebased now that #76708 landed.